pkg/glusterfs/deploy/kubernetes/csi-nodeplugin-glusterfsplugin.yaml

```diff
@@ -0,0 +1,81 @@
+# Copyright 2018 The Gluster CSI Authors.
```

Comment:
Hang on! copyright here and elsewhere was k8s authors, and we just change it? We possibly need to understand content copied and modified to see which copyright to retain, aka I am no lawyer!

Reply:
This was my suggestion (from https://github.com/gluster/gluster-csi-driver/pull/1#issuecomment-369738696). New content gets "Gluster CSI", but copied stuff retains the old, and modified should have both.

Reply:
Ack! So this is new content, right @humblec ?

Reply:
yes, thats correct Shyam!